tests/Neo.CLI.Tests/UT_MainService_Contracts.cs

```diff
@@ -0,0 +1,576 @@
+                    {
+                        new ContractParameterDefinition { Name = "value", Type = ContractParameterType.Hash160 }
+                    },
+                    ReturnType = ContractParameterType.Hash160,
+                    Safe = true
+                },
+                new ContractMethodDescriptor
+                {
+                    Name = "testArray",
+                    Parameters = new ContractParameterDefinition[]
+                    {
+                        new ContractParameterDefinition { Name = "values", Type = ContractParameterType.Array }
+                    },
+                    ReturnType = ContractParameterType.Array,
+                    Safe = true
+                },
+                new ContractMethodDescriptor
+                {
+                    Name = "testMultipleParams",
+                    Parameters = new ContractParameterDefinition[]
+                    {
+                        new ContractParameterDefinition { Name = "from", Type = ContractParameterType.Hash160 },
+                        new ContractParameterDefinition { Name = "to", Type = ContractParameterType.Hash160 },
+                        new ContractParameterDefinition { Name = "amount", Type = ContractParameterType.Integer }
+                    },
+                    ReturnType = ContractParameterType.Boolean,
+                    Safe = false
+                }
+            };
+
+            manifest.Abi.Methods = methods.ToArray();
+
+            // Create a simple contract script
+            using var sb = new ScriptBuilder();
+            sb.EmitPush(true);
+            sb.Emit(OpCode.RET);
+            var script = sb.ToArray();
+
+            // Create NefFile
+            var nef = new NefFile
+            {
+                Compiler = "",
+                Source = "",
+                Tokens = Array.Empty<MethodToken>(),
+                Script = script
+            };
+            nef.CheckSum = NefFile.ComputeChecksum(nef);
+
+            // Create the contract state manually
+            _contractState = new ContractState
+            {
+                Id = 1,
+                Hash = _contractHash,
+                Nef = nef,
+                Manifest = manifest
+            };
+
+            // Properly add the contract to the test snapshot using the extension method
+            var snapshot = _neoSystem.GetSnapshotCache();
+            snapshot.AddContract(_contractHash, _contractState);
+
+            // Commit the changes to make them available for subsequent operations
+            snapshot.Commit();
+        }
+
+        [TestMethod]
+        public void TestParseParameterFromAbi_Boolean()
+        {
+            var method = GetPrivateMethod("ParseParameterFromAbi");
+
+            // Test true value
+            var result = (ContractParameter)method.Invoke(_mainService, new object[] { ContractParameterType.Boolean, JToken.Parse("true") });
```

Comment:
```suggestion
            var result = (ContractParameter)method.Invoke(_mainService, [ContractParameterType.Boolean, JToken.Parse("true")]);
```
use collection expresstion(this and others)